enable/trait_defs/ui/kiva_font_editor.py

```diff
@@ -0,0 +1,128 @@
+# (C) Copyright 2005-2022 Enthought, Inc., Austin, TX
+# All rights reserved.
+#
+# This software is provided without warranty under the terms of the BSD
+# license included in LICENSE.txt and may be redistributed only under
+# the conditions described in the aforementioned license. The license
+# is also available online at http://www.enthought.com/licenses/BSD.txt
+#
+# Thanks for using Enthought open source!
+
+from pyface.font import Font as PyfaceFont
+from pyface.font_dialog import get_font
+from traits.api import Bool, Callable, Instance, Str, observe
+from traits.trait_base import SequenceTypes
+from traitsui.api import EditorFactory, Editor as BaseEditor, toolkit_object
+
+from kiva.fonttools.font import Font
+import kiva.constants as kc
+from enable.component import Component
+from enable.label import Label
+from enable.tools.button_tool import ButtonTool
+from enable.window import Window
+from .editor_with_component import EditorWithLabelComponent
+
+
+def face_name(font):
+    """ Returns a Font's typeface name.
+    """
+    face_name = font.face_name
+    if isinstance(face_name, SequenceTypes):
+        face_name = face_name[0]
```

Comment:
Is this branch exercised? The `Font` constructor seems to enforce that `face_name` is always an instance of `str`, and I don't seem to hit this branch at all when running the test suite. (If the branch _is_ needed, it might be good to have a test that exercises it.)

Reply:
This was just copied over from the previous implementation.

I _think_ what this allows is that the function can be used on `Font`-like objects that can specify "if we don't have font X, font Y is acceptable" via a list of font names (eg. as the Pyface `Font` does, although this code predates that class).  So I don't want to just change it now, but I'll add an issue.